scripts/utils/validateNode.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019
+################################################################################
+
+# NODE_HOME Should contain a valid install of Node
```

Comment:
Consider checking for the correct version in this script

Reply:
add minimum version check of 6.00, not checking